src/content/docs/en/guides/fonts.mdx

```diff
@@ -9,22 +9,22 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 
 
 
-Astro supports all common strategies for adding custom typefaces to your site design. This guide will show you two different options for including web fonts in your project.
+Astro supports all common strategies for adding custom typefaces to your site design. This guide will show you two options for including web fonts in your project.
 
-## Using a local font file
+## Add fonts to your project
```

Comment:
I'm not sure about this edit to have two top-level headings : "Add fonts to your project" and "Integrations". 
- "Add fonts to your project" actually shows you both how to add the fonts (put them in `public`; install with Fontsource) and how to use them (`font-family`)
- The "Integrations" title isn't very descriptive 
- There's only one subsection there, and I can't think of other potential subsections
- The Tailwind subsection relies on the other two subsections, implying that they're in a sequence. Separating it into another larger section makes it seem independent. 

My suggestion: "Add fonts with Tailwind" to "Register fonts in Tailwind" and have all of the sections top-level.

Happy to defer to @sarah11918 on this.

Reply:
Well, that's a good one.

Initially, in the intro there was a phrase telling that there are two options of including fonts to your project but when you are checking the sidebar, there were 3 of them, in fact, including Tailwind integration which was a bit confusing. So I decided to split those parts. As long as we revised the intro in this PR as well, I have no objections about keeping the Tailwind section as before, following your suggestion.

Reply:
I suggest switching to an even simpler document structure

Before | After
-------|------
<img width="224" alt="toc" src="https://user-images.githubusercontent.com/16765690/221992389-0ef6cbdf-3dae-494d-8f26-2f159a553572.png"> | <img width="215" alt="toc2" src="https://user-images.githubusercontent.com/16765690/221993483-7d73c35e-f2ee-4739-985b-eafb75db4db4.png">

@Jutanium @sarah11918 What do you think?

Reply:
@bandantonio Agreed, this is what I had in mind!